client/docs/config.json

```diff
@@ -13,79 +13,79 @@
       }
     },
     {
-      "url": "./tmp-swagger-gen/cosmos/auth/v1beta1/query.swagger.json",
```

Comment:
How about change the title of swagger info above?
`Cosmos SDK - Legacy ~~~` -> ` LFB SDK - Legacy`

Reply:
thks! changed it